### DISCUSSION TITLE
`git spr` for stacked pull requests

### DISCUSSION THREAD

**Discussion by arxanas** [gitext-rs/git-stack#186] [General]
New tool, might be worth examining: https://github.com/ejoffe/spr

**Comment by epage** [comment]
Thanks for sharing this!

Its unfortunate that they don't document how stacks are represented within Github (e.g. merging into each others branches, all merging into target branch, special branch setup like `ghstack`, etc) or what the PRs look like (e.g. `gh-stack` showing other parts of stack)

**Reply by epage** [reply]
Oh and I have added it to the comparison despite not being able to list many details
